assets/install-scripts/install-selinux.sh

```diff
@@ -36,21 +41,23 @@ while getopts "c:t:" opt; do
 done
 
 # Write SELinux module source to a temporary directory
-WORK_DIR="$(mktemp -d)"
+WORK_DIR="$(mktemp -d teleport-selinux.XXXXXXXX)"
```

Comment:
This will create a directory in the current working directory. You _probably_ want `mktemp -d --suffix .teleport-selinux` or something to that effect:

```console
root@4c736d6b6b90:/host# mktemp -d --suffix .teleport-selinux
/tmp/tmp.sJlWRvFzvY.teleport-selinux
```

Reply:
Ah thanks, was a bit confused by that; changed